tests/cases/standalone/common/select/system.result

```diff
@@ -0,0 +1,10 @@
+-- SQLNESS REPLACE branch:\s+.+ branch: BRANCH
+-- SQLNESS REPLACE commit:\s+.+ commit: COMMIT
+-- SQLNESS REPLACE commit+\s+short:\s+.+ commit short: COMMIT SHORT
+-- SQLNESS REPLACE dirty:\s+.+ dirty: DIRTY
+-- SQLNESS REPLACE version:\s+.+ version: VERSION
+-- SQLNESS REPLACE [\s\-]+
+SELECT build();
```

Comment:
Can we move the `system.sql` and `system.result` to `tests/cases/standalone/common/function`? It's better to put them there.

Reply:
Sure